components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,1223 @@
+
+momentum:
+
+$$
+\frac{\partial }{\partial t} \int_{s_k^{bot}}^{s_k^{top}} \rho g{\bf u} dz
++  \int_{s_k^{bot}}^{s_k^{top}} \rho g \eta {\bf k} \times {\bf u} dz
++  \int_{s_k^{bot}}^{s_k^{top}} \rho g w\frac{\partial {\bf u}}{\partial z} dz
+  =  \int_{s_k^{bot}}^{s_k^{top}} \left[- \frac{1}{\rho}\nabla p - \nabla K
++ {\bf D}^u + {\bf F}^u \right] dz
+$$ (layer-int-momentum-R13App)
+
+mass:
+
+$$
+\frac{\partial }{\partial t}  \int_{s_k^{bot}}^{s_k^{top}} \rho g dz
+ +\nabla \cdot \left( \int_{s_k^{bot}}^{s_k^{top}} \rho g  {\bf u} dz \right)
+ + \int_{s_k^{bot}}^{s_k^{top}}\frac{\partial}{\partial z} \left(  \rho g  w \right) dz
+= \int_{s_k^{bot}}^{s_k^{top}} S^h dz
+$$ (layer-int-mass-R13App)
+
+tracer:
+
+$$
+\frac{\partial }{\partial t} \int_{s_k^{bot}}^{s_k^{top}} \rho g \varphi  dz
+ + \nabla \cdot \left( \int_{s_k^{bot}}^{s_k^{top}} \rho g \varphi {\bf u} dz  \right)
+ + \int_{s_k^{bot}}^{s_k^{top}} \frac{\partial}{\partial z} \left(  \rho g \varphi w \right) dz
+= \int_{s_k^{bot}}^{s_k^{top}} \left( D^\varphi + S^\varphi \right) dz
+$$ (layer-int-tracer-R13App)
+
+Equivalently, these equations may be written as integrating in pressure in the vertical from $\sigma_k^{bot}$ to $\sigma_k^{top}$:
+
+momentum:
+
+$$
+\frac{\partial }{\partial t} \int_{\sigma_k^{bot}}^{\sigma_k^{top}} {\bf u} dp
++  \int_{\sigma_k^{bot}}^{\sigma_k^{top}}  \eta {\bf k} \times {\bf u} dp
++  \int_{\sigma_k^{bot}}^{\sigma_k^{top}}  w\frac{\partial {\bf u}}{\partial z} dp
+  =  \int_{\sigma_k^{bot}}^{\sigma_k^{top}} \left[ - \frac{1}{\rho}\nabla p - \nabla K
++ {\bf D}^u + {\bf F}^u \right] dp
+$$ (layer-int-momentum-R13App)
+
+mass:
+
+$$
+\frac{\partial }{\partial t}  \int_{s_k^{bot}}^{s_k^{top}}  dp
+ +\nabla \cdot \left( \int_{s_k^{bot}}^{s_k^{top}}   {\bf u} dp \right)
+ + \int_{s_k^{bot}}^{s_k^{top}}\frac{\partial}{\partial z} \left(    w \right) dp
+= \int_{s_k^{bot}}^{s_k^{top}} S^h dp
+$$ (layer-int-mass-R13App)
+
+tracer:
+
+$$
+\frac{\partial }{\partial t} \int_{s_k^{bot}}^{s_k^{top}}  \varphi  dp
+ + \nabla \cdot \left( \int_{s_k^{bot}}^{s_k^{top}}  \varphi {\bf u} dp  \right)
+ + \int_{s_k^{bot}}^{s_k^{top}} \frac{\partial}{\partial z} \left(   \varphi w \right) dp
+= \int_{s_k^{bot}}^{s_k^{top}} \left( D^\varphi + S^\varphi \right) dp
+$$ (layer-int-tracer-R13App)
+
+
+Rewriting the conservation equations in with these layer-averaged variables results in the final layered equations,
+
+$$
+\frac{\partial \mathbf{u}_k}{\partial t}
++ q_k\left(h\mathbf{u}_k^{\perp}\right) 
++ \nabla K_k
+= 
+- \frac{1}{\rho} \nabla_r p - \nabla_r \phi
++ {\bf D}_k^u + {\bf F}_k^u
+$$ (layered-momentum)
+
+$$
+\frac{\partial h_k}{\partial t} + \nabla_r \cdot \left(h_k \mathbf{u}_k\right) + \omega_{bot} - \omega_{top}= 0,
+$$ (layered-mass)
+
+$$
+\frac{\partial h_k \varphi_k}{\partial t} + \nabla_r \cdot \left(h_k \mathbf{u}_k \varphi_k\right) 
++ \varphi\omega_{bot} - \varphi\omega_{top} 
+=  D^\varphi_k + S^\varphi_k 
+$$ (layered-tracer)
+
+<!--
```

Comment:
Again, let's please remove commented-out stuff for ease of reveiw.

Reply:
done.